ee/vellum_cli/tests/test_pull.py

```diff
@@ -25,7 +26,15 @@ def _zip_file_map(file_map: dict[str, str]) -> bytes:
     return zip_bytes
 
 
-def test_pull(vellum_client, mock_module):
+@pytest.mark.parametrize(
+    "base_command",
+    [
+        ["pull"],
+        ["pull", "workflows"],
+    ],
+    ids=["pull", "pull_workflows"],
```

Comment:
implemented this pattern in the push tests below, decided to apply it to a pull test as well